_includes/important.html

```diff
@@ -0,0 +1,3 @@
+<div markdown="span" class="alert alert-warning" role="alert"><i class="fa fa-warning"></i> <b>Important:</b> {{include.content}}</div>
+
+{% comment %} Copyright 2021 Google LLC {% endcomment %}
```

Comment:
what is the legal implication of having these Copyright notices?

Reply:
The original that those came from is ASL, so we're free to use it as we choose, but we're not free to remove an existing copyright.